### DISCUSSION TITLE
Silently Copying a Repo to Your GitHub Without Forking

### DISCUSSION THREAD

**Discussion by romeouno** [community/community#145608] [Repositories]
Hey everyone,

I wanted to start a discussion about copying a GitHub repository without using the usual "fork" button. Forking is awesome for contributing to a project or creating a copy that stays linked to the original repo. But there are times when you might just want a fresh copy of a repo—without that "fork" relationship.

**Comment by AuroraEchos** [comment]
Hey,bro, I think you can try these methods to solve your problem. 
1. use **git clone --mirror**,This method clones a mirror repository, including all branches, labels, and other references. 
2. Use **git clone --bare**,This method clones a raw repository that contains only Git's internal data and no working directory.
3. Use **the normal git clone command** to clone the original repository, and you will get a complete repository containing the working directory.

**Comment by DZAKYALR** [comment]
Try this
`git clone --mirror <link repo>`

This command creates an exact copy of the repository, including all branches, tags, and references. It’s ideal for backups or migrating a repository to a new server.

**Comment by tuves** [comment]
Welcome to the GitHub Community, @romeouno, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
